src/services/activity.js

```diff
@@ -32,6 +48,42 @@ const getAll = sessionCode =>
     .get(`activities/session/${sessionCode}`)
     .then(activities => sortBy(activities, activity => activity.ActivityDescription));
 
+/**
+ * Get number of followers of an activity
+ * @param {String} activityCode Identifier for an activity
+ * @param {String} sessionCode Identifier for a session
+ * @returns {Number} Number of followers
+ */
+const getFollowersNum = (activityCode, sessionCode) =>
+  http.get(`memberships/activity/${activityCode}/followers/${sessionCode}`);
+
+/**
+ * Get group administrators for an activity
+ * @param {String} activityCode Identifier for an activity
+ * @param {String} sessionCode Identifier for a session
+ * @returns {Person[]} List of group administrators
```

Comment:
What does `Person` refer to? In order to use a JSDoc type, you must first define that type.

Look at the docs for [`@typedef`](http://usejsdoc.org/tags-typedef.html) to learn more. Use the type definitions in the other services for examples.